test/test.js

```diff
@@ -158,6 +158,46 @@ describe('remapify', function(){
     }])
   })
 
+  it('looks for files in multiple directories, in precedence', function(done){
+    b.on('remapify:files', function(files, expandedAliases){
+      expandedAliases.should.contain.keys(
+        'a.js',
+        'b.js'
+      )
+      expandedAliases['a.js'].split(path.sep).join('/').should.equal('./test/fixtures/target2/a.js')
+      expandedAliases['b.js'].split(path.sep).join('/').should.equal('./test/fixtures/target/b.js')
+
+      b.emit.should.not.have.been.calledWith('error')
+
+      done()
+    })
+
+    plugin(b, [{
+      src: '**/*.js'
+      , cwd: ['./test/fixtures/target2', './test/fixtures/target']
+    }])
+  })
+
+  it('multiple cwds works when passing in an array from command line', function(done){
+    b.on('remapify:files', function(files, expandedAliases){
+      expandedAliases.should.contain.keys(
+        'a.js',
+        'b.js'
+      )
+      expandedAliases['a.js'].split(path.sep).join('/').should.equal('./test/fixtures/target2/a.js')
+      expandedAliases['b.js'].split(path.sep).join('/').should.equal('./test/fixtures/target/b.js')
+
+      b.emit.should.not.have.been.calledWith('error')
+
+      done()
+    })
+
+    plugin(b, [{
+      src: '**/*.js'
+      , cwd: {'_': ['./test/fixtures/target2', './test/fixtures/target'] }
```

Comment:
Yeah, it does accept an array, but I'm also accepting a map with one key. This is because browserify seems to call remapify with the above arguments, if called as follows from the command line:

```browserify -p [ reactify --cwd ['/directory1' '/directory2'] ]```

I have no idea if this is documented to work like this, or if it's random, but it does this consistently.

Thus we accept 4 types of arguments for cwd
* null / undefined => processCwd
* a string => the actual CWD
* an array
* the above structure from command line